### PR TITLE
Add extension test infra with a simple initial page extractor test.

### PR DESCRIPTION
**Pull request by shaper**
I noted the ordered list prefixes have to have '.' to format properly at least in the Markdown extension I'm using to read these in Chrome:

https://chrome.google.com/webstore/detail/gpoigdifkoadgajcincpilkjmejcaanc

### CODE REVIEW

**Comment by shaper** [comment]
Adding @sjmiles as well so that he can see what I ended up doing (`mocha-chrome`) to allow running via `npm test`. We could do similar in `arcs-cdn`-land.

File: README.md [hunk 1]
```diff
@@ -13,12 +13,14 @@ $ (cd tracelib && npm install)
...
 $ (cd runtime && npm install)
 $ (cd strategy-explorer && bower install)
+$ (cd extension && npm install)
 
 ```
 
 ## Test
 ```
 $ (cd runtime && gulp)
+$ (cd extension && npm test)
```
Comment (smalls) [inline README.md]: Would 'npm test' work for both runtime & the extension (or gulp work for both)?

The more consistent we can be the less surprising we'll be to new developers.

Comment (shaper) [reply README.md]: It looks like 'npm test' will work for both, so will revise accordingly. I'm seeing a couple of tests fail right now at ToT but that seems to be the case with both 'gulp' and 'npm test'.

**Comment by shaper** [comment]
https://github.com/PolymerLabs/arcs/pull/512 is the result of the manual CLI notes.

**Comment by shaper** [comment]
Closing in favor of https://github.com/PolymerLabs/arcs/pull/514

**Pull request closed without merging**